postgres/messages/row_description.go

```diff
@@ -23,6 +23,87 @@ import (
 	"github.com/dolthub/doltgresql/postgres/connection"
 )
 
+const (
+	OidBool        = 16
+	OidBytea       = 17
+	OidChar        = 18
+	OidName        = 19
+	OidInt8        = 20
+	OidInt2        = 21
+	OidInt2Vector  = 22
+	OidInt4        = 23
+	OidRegproc     = 24
+	OidText        = 25
+	OidOid         = 26
+	OidTid         = 27
+	OidXid         = 28
+	OidCid         = 29
+	OidOidVector   = 30
+	OidPgNodeTree  = 194
+	OidPgString    = 1033
```

Comment:
I'd order these by number or by name, cause otherwise it seems mostly by number but not quite all the way there, and it's a little confusing lol